examples/open_llama/llama_qlora.json

```diff
@@ -0,0 +1,76 @@
+{
+    "input_model":{
+        "type": "PyTorchModel",
+        "config": {
+            "hf_config": {
+                "model_name": "huggyllama/llama-7b",
```

Comment:
Any reason? why we do not use `openlm-research/open_llama_7b`？

Reply:
I wanted to replicate the example from the original implementation https://github.com/artidoro/qlora/blob/main/scripts/finetune_guanaco_7b.sh